packages/backend/src/core/activity/counters/StarkexCounter.ts

```diff
@@ -62,5 +62,5 @@ export function createStarkexCounter(
 }
 
 function getStarkexLastDay(timestamp: UnixTime) {
-  return timestamp.toStartOf('day').add(-1, 'days').toDays()
+  return timestamp.toStartOf('day').add(-1, 'days').add(-4, 'hours').toDays() // StarkEx APIs have inherent delay
```

Comment:
This fix looks great. I only wonder if we could make this offset even bigger? I am not aware about the amount of computation that it will include, so maybe it is not the best idea. But I am also worried that some day they change the delay and our code will stop working.

Reply:
This does not increase the amount of computation, it just makes StarkexSequenceProcessors start the sync 4 hours later.

Reply:
This will make update to all processors delayed by 4 hours aka we will show new data on 4am in the morning. That's why I want to keep this value minimal.